ci/ec2_products_cost.json

```diff
@@ -4,5 +4,6 @@
   "p3.2xlarge": 3.06,
   "p4d.24xlarge": 32.7726,
   "p5.48xlarge": 98.32,
-  "rtx4090": 0.04
+  "rtx4090": 0.04,
+  "single-h100": 4.30
```

Comment:
is single h100 the 'official' name from hyperstack ?

Reply:
No it's not. I can replace with the official name if you prefer.

Reply:
yes this way it's easier to find which provider it comes from